spec/gollum_git_commit_spec.rb

```diff
@@ -38,4 +39,7 @@
     expect(commit.tree).to be_a Gollum::Git::Tree
   end
 
+  it "returns a Gollum::Git::Commit object for parent" do
+    expect(commit.parent).to be_a Gollum::Git::Commit
```

Comment:
What if there is no parent?

Reply:
Then it should return `nil`, of course :) But how could we test this in the specs?

Reply:
Something like `expect(repo.commits.first.parent).to eq nil`?

Reply:
Yes, but you'll either have to use mocking to get the `Rugged::Commit`'s parents to be empty, or find a commit for which that's the case in the test repository

Reply:
Sorry I don't get the point. I thought `repo.commits.first` always returns the root commit with no parent.

Reply:
eh, I guess that's right. Ignore me 😆